ivy/functional/backends/torch/statistical.py

```diff
@@ -265,6 +265,42 @@ def cumprod(
 cumprod.support_native_out = True
 
 
+# my implementation. remove comment before commiting
+# what do exclusive and reverse do?
+# what do with unsupposrted_dtypes do?
+# how are these unsupported datatypes figured out?
```

Comment:
TensorFlow provides decent documentation explaining what `exclusive` and `reverse` do, check it out [here](https://www.tensorflow.org/api_docs/python/tf/math/cumprod)

Unsupported dtypes are the dtypes excluded from this function, we don't generate tests using these dtypes in the testing helpers. In other words, the function is not meant to be used with such dtypes. You can ask about that in the `ivy-tests` channel on discord. :)

Reply:
I added these comments when I first started to learn about ivy. I forgot to remove them. Let me fix it and request re-review
this is my first PR. I spent a lot of time on getting the compositional functions right. I completely forgot about these little comments and doc strings by the end of it. Let me fix it and request a re-review